project/Tests/Snapper.Tests/SnapperNewtonsoftNuisancesTests.cs

```diff
@@ -0,0 +1,39 @@
+using System.Runtime.CompilerServices;
+using Snapper.Attributes;
+using Xunit;
+
+namespace Snapper.Tests
+{
+    /// <summary>
+    ///     This class contains tests which test that the nuisances of newtonsoft are avoided in Snapper
+    /// </summary>
+    [StoreSnapshotsPerClass]
+    public class SnapperNewtonsoftNuisancesTests
+    {
+        [Fact]
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        // Related issue https://github.com/JamesNK/Newtonsoft.Json/issues/862
+        public void DateTimeIsParsedAsStringBySnapper_FileSnapshot()
+        {
+            const string snapshot = "{" +
```

Comment:
Not sure how much of a difference it makes but usually snapshots are tested against objects rather than strings so alternatively `snapshot` could be declared e.g. like this:

```
var snapshot = new
            {
                Key = new DateTimeOffset(2010, 12, 31, 0, 0, 0, TimeSpan.FromHours(10))
            };
```
but like I said, not sure how relevant that is :)

Reply:
Good point, I've added tests for that too since they go through a slightly different path.